android/build.gradle

```diff
@@ -122,4 +129,5 @@ hyperSdkPlugin {
     clientId = getClientId(rootProject)
     sdkVersion = getHyperSDKVersion(rootProject)
     excludedMicroSDKs = getExcludedMicroSdks(rootProject)
+    hyperAssetVersion = getHyperAssetVersion(rootProject)
```

Comment:
@sarfarazsheikh10 `hyperAssetVersion` does not exist in `hyperSdkPlugin`. Change this to `assetVersion`

Reply:
+1